server/controllers/v1/shows/findAll.js

```diff
@@ -1,7 +1,12 @@
 const db = require('../../../models/v1');
+const keys = require('../../../config/keys');
```

Comment:
NOTE: the changes to this file will be written by some of Josh's changes later. They are only here now so the app doesn't freeze when navigating to the Show Calendar page.